jcl/src/jdk.management/share/classes/openj9/lang/management/OpenJ9DiagnosticsMXBean.java

```diff
@@ -0,0 +1,147 @@
+	 * @throws SecurityException if there is a security manager and it doesn't allow the checks required to change the dump settings
+	 * @throws NullPointerException if options is null
+	 */
+	public void setDumpOptions(String dumpOptions) throws InvalidDumpOptionException, DumpConfigurationUnavailableException;
+
+	/**
+	 * This function triggers the specified dump agent. Dump agents supported - java, snap, system, heap
+	 * A java dump is in a human-readable format, and summarizes the state of the JVM
+         * The default heap dump format (a phd file) is not human-readable.
+         * A system dump is a platform-specific file that contains information about the active processes, threads, and
+	 * system memory. System dumps are usually large. 
+	 * The snap dump format is not human-readable and must be processed using the trace formatting tool supplied  with the OpenJ9 JVM.
+	 *
+	 * @param dumpAgent the dump agent to be triggered
+	 * @throws IllegalArgumentException if the specified dump agent is invalid or unsupported by this method
+	 * @throws RuntimeException if the vm does not contain RAS dump support
+	 * @throws SecurityException if there is a security manager and it doesn't allow the checks required to trigger this dump
+	 */
+	public void triggerDump(String dumpAgent) throws IllegalArgumentException;
+
+	/**
+	 * This function triggers the specified dump agent. Dump agents supported - java, snap, system, heap.
+	 * The JVM will attempt to write the file to the specified file name. This may
+	 * include replacement tokens as documented in the section on dump agents
+	 * in the documentation for the OpenJ9 JVM.
+	 * 
+	 * A string containing the actual file name written to is returned. This may not
+	 * be the same as the requested filename for several reasons:
+	 * <ul>
+	 * <li>null or the empty string was specified, this will cause the JVM to write the
+	 *  dump to the default location based on the current dump settings and return that
+	 *  path.</li>
+	 * <li>Replacement (%) tokens were specified in the file name. These will have been
+	 *  expanded.</li>
+	 * <li>The full path is returned, if only a file name with no directory was specified
+	 *  the full path with the directory the dump was written to will be returned.</li>
+	 * <li>The JVM couldn't write to the specified location. In this case it will attempt
+	 *  to write the dump to another location, unless -Xdump:nofailover was specified on
+	 *  the command line.</li>
+	 * </ul>
+	 * 
+	 * If a security manager exists a permission check for com.ibm.jvm.DumpPermission will be
+	 * made, if this fails a SecurityException will be thrown.
+	 *
+	 * @return the file name that the dump was actually written to 
+	 * @param dumpAgent the dump agent to be triggered
+	 * @param fileNamePattern the file name to write to, which may be null, empty or include replacement tokens
+	 * @throws InvalidDumpOptionException if the filename was invalid
```

Comment:
This should refer to 'fileNamePattern' instead of 'filename'.
Please be consistent: 'file name' or 'filename', but not both.

Reply:
Took the description from Dump.java, will make changes there as well